src/js/components/productSupplier/create/subsections/DefaultPreferenceType.jsx

```diff
@@ -1,21 +1,54 @@
 import React from 'react';
 
 import PropTypes from 'prop-types';
-import { Controller } from 'react-hook-form';
+import { Controller, useWatch } from 'react-hook-form';
+import { RiDeleteBinLine } from 'react-icons/ri';
 import { useSelector } from 'react-redux';
+import { Tooltip } from 'react-tippy';
 
 import DateField from 'components/form-elements/v2/DateField';
 import SelectField from 'components/form-elements/v2/SelectField';
 import TextInput from 'components/form-elements/v2/TextInput';
 import Subsection from 'components/Layout/v2/Subsection';
+import useDeletePreferenceType from 'hooks/productSupplier/form/useDeletePreferenceType';
+import Translate from 'utils/Translate';
 
-const DefaultPreferenceType = ({ control, errors }) => {
+const DefaultPreferenceType = ({
+  control,
+  errors,
+  setValue,
+}) => {
   const {
     preferenceTypes,
   } = useSelector((state) => ({
     preferenceTypes: state.productSupplier.preferenceTypes,
   }));
 
+  const updatedDefaultPreferenceType = useWatch({
+    name: 'defaultPreferenceType',
+    control,
+  });
+
+  const emptyPreferenceType = {
+    bidName: '',
+    validityStartDate: '',
+    validityEndDate: '',
+    preferenceType: '',
+  };
+
+  const afterDelete = () => {
+    setValue('defaultPreferenceType', emptyPreferenceType);
+  };
```

Comment:
not a must have, but I just feel like 27-41 lines could also be included in the hook you've created.
Another not "must have" (because it won't impact the performance at all) is that the `emptyPreferenceType` could be outside of the component, because its body never changes when a re-render happens (it is not dependent on the component life cycle)